### PR TITLE
Fix RJSF stale state issue

### PR DESCRIPTION
**Pull request by tangledbytes**
**Description**

This PR fixes stale state issue in the pattern service handler

**Notes for Reviewers**
Previous hacks are now removed :tada: 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by utkarsh-pro** [comment]
@killbotXD here is the fix

**Review by killbotXD — APPROVED** [review]
Lgtm. Thanks mate

File: ui/components/MesheryMeshInterface/PatternServiceForm.js [hunk 1]
```diff
@@ -63,15 +64,18 @@ function createPatternFromConfig(config, namespace) {
 
   Object.keys(config).forEach((key) => {
     // Add it only if the settings are non empty or "true"
-    if (config[key].settings) pattern.services[key] = config[key];
+    if (config[key].settings) {
+      const name = PascalCaseToKebab(key);
```
Comment (killbotXD) [inline ui/components/MesheryMeshInterface/PatternServiceForm.js:68]: this is for the service name right?

Comment (utkarsh-pro) [reply ui/components/MesheryMeshInterface/PatternServiceForm.js:68]: Yes. This should fix the Virtual Service issue

**Comment by leecalcote** [comment]
Hacks-be-gone!

**Review by leecalcote — APPROVED** [review]

**Pull request merged**